src/include/txn.h

```diff
@@ -42,9 +43,10 @@
 /* AUTOMATIC FLAG VALUE GENERATION STOP 32 */
 
 typedef enum {
-    WT_VISIBLE_FALSE = 0,   /* Not a visible update */
-    WT_VISIBLE_PREPARE = 1, /* Prepared update */
-    WT_VISIBLE_TRUE = 2     /* A visible update */
+    WT_VISIBLE_FALSE = 0,      /* Not a visible update */
+    WT_VISIBLE_PREPARE = 1,    /* Prepared update */
+    WT_VISIBLE_NONDURABLE = 2, /* Committed but not durable update */
+    WT_VISIBLE_TRUE = 3        /* A visible update */
```

Comment:
Sort order, please. There's no reason to initialize these enums either, but not a problem either.

Reply:
Oops, fixed.